internal/engine/eval/rego/result.go

```diff
@@ -170,26 +172,26 @@ func (*constraintsEvaluator) getQuery() func(r *rego.Rego) {
 	return rego.Query(fmt.Sprintf("%s.violations[details]", RegoQueryPrefix))
 }
 
-func (c *constraintsEvaluator) parseResult(rs rego.ResultSet, _ protoreflect.ProtoMessage) error {
+func (c *constraintsEvaluator) parseResult(rs rego.ResultSet, _ protoreflect.ProtoMessage) (*interfaces.EvaluationResult, error) {
 	if len(rs) == 0 {
 		// There were no violations
-		return nil
+		return &interfaces.EvaluationResult{}, nil
```

Comment:
Would it make sense to have `Output: []any{}` here, or will Output being `nil` be handled the same as an empty list?

Reply:
It will be handled the same as an empty list. I tested it in go code and in the template rendering, is there some other scenario I should check?

Reply:
No, this was just a question I was too lazy to check myself.  😁